es/installation/debian.md

```diff
@@ -39,7 +39,7 @@ apt-get install curl
 
 ## Ruby
 
-Las versiones de Ruby versions empaquetadas en repositorios oficiales no son aptas para trabajar con consul (al menos Debian 7 y 8), así que debemos instalar manualmente.
+Las versiones de Ruby versions empaquetadas en repositorios oficiales no son aptas para trabajar con consul, así que debemos instalar manualmente.
```

Comment:
Since we're changing this line, how about fixing a couple of typos here?

1. "Las versiones de Ruby empaquetadas" instead of "Las versiones de Ruby versions empaquetadas"
2. Add a pronoun either before of after "debemos instalar"